frontend/components/artists/artist_page_menu_bar.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from "react";
+import ArtistPageDropdown from "./artist_page_dropdown";
+
+import { RxDotsHorizontal } from "react-icons/rx";
+import { GrPlayFill } from "react-icons/gr";
+
+const ArtistPageMenuBar = ({
+	currentArtist,
+	history,
+	fetchPlaylists,
+	editPlaylist,
+}) => {
+	const [artistPageDropdownState, setArtistPageDropdownState] = useState({
+		isOpen: false,
```

Comment:
curious why you gave this an entire object?

Reply:
Oh wow, I think this may have been an aaccidental auto-complete. Thanks for noticing.

Reply:
I just think it makes it more readable. `artistPageDropdownState.isOpen?` reads slightly clearer than just `artistPageDropdownState?`